exporter/containerimage/writer.go

```diff
@@ -177,6 +185,162 @@ func (ic *ImageWriter) exportLayers(ctx context.Context, compressionType compres
 	return out, nil
 }
 
+func (ic *ImageWriter) ensureCompressionType(ctx context.Context, remote *solver.Remote, forceCompressionType compression.Type) (*solver.Remote, error) {
```

Comment:
Can we use containerd converter API as in `ctr-remote image convert` and `nerdctl image convert`

Reply:
Updated to use converter API.